docs/book/content/contributing/contributor_guide.md

```diff
@@ -72,19 +51,19 @@ installation that came as part of your computer's operating system.
 
 10. The prior command will create a conda environment called `ogusa-dev`.
     Activate this environment as follows:
+
     ```
-      OG-USA$ source activate ogusa-dev
+      OG-USA$ conda activate ogusa-dev
     ```
-    If you are working on Windows, use the following from the command line::
+    If you are working on Windows, use the following from the command line:
```

Comment:
These separate instructions for Windows can be deleted. Now all OS's use `conda activate`.

Reply:
Removed separate Windows OS instruction. Thanks @jdebacker